autoPyTorch/pipeline/tabular_classification.py

```diff
@@ -225,6 +227,39 @@ def _get_pipeline_steps(self, dataset_properties: Optional[Dict[str, Any]],
         ])
         return steps
 
+    def get_pipeline_representation(self) -> Dict[str, str]:
+        """
+        Returns a representation of the pipeline, so that it can be
+        consumed and formatted by the API.
+
+        It should be a representation that follows:
+        [{'PreProcessing': <>, 'Estimator': <>}]
+
+        Returns:
+            Dict: contains the pipeline representation in a short format
+        """
+        preprocessing = []
+        estimator = []
+        skip_steps = ['data_loader', 'trainer', 'lr_scheduler', 'optimizer', 'network_init',
```

Comment:
Maybe we can have a verbose option where we include- trainer, lr_scheduler, optimizer and network_init?

Reply:
I was coding this and seeing the outcome, and it just doesn't add much information to 'what the best estimator is' because it's part of the construction of the model, but not part of the model itself.

I have a better proposal -- I would like to create a command like [this](https://github.com/EpistasisLab/tpot/blob/6448bdb71ba08b4a0447c640d2f05a05e1affc21/tpot/export_utils.py#L56).

Tpot is able to print the python code on how to train the model that you did. So if your goal were to see what happened (and for debug purposes) it would be great to produce a file that contains PyTorch commands with not only the scheduler but also the config (like it is nice to know we know more that the fact that we picked `adam` optimizers). If this is better, then I would like to disentangle this export_pipeline command from this PR and create an issue for this.
What do you think?

Reply:
yes this would be ideal to use. Could you add an issue so we don't forget?